docs/mine/lotus/miner-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+
+## Restarting workers
+
+Lotus [seal workers](seal-workers.md) can be restarted any time, but if they are in the middle of one of the sealing steps, then the operation will start again (from the last checkpoint).
+
+::: warning
+There is a maximum of three attempts to complete a sealing operation before it is fully started from scratch (_pre-commit1_ phase).
+:::
+
+## Changing storage locations
+
+If you wish to change the location of your miner-related storage to a different path, for the miner or the seal workers, make sure that the Lotus miner and any seal workers are aware of the new location.
+
+```sh
+lotus-miner storage list
+```
+
+The above command will give you an overview of [storage locations known to the miner](custom-storage-layout.md). This information is stored in `~/lotusminer/storage.json` (or `$LOTUS_MINER_PATH/storage.json` if defined). Lotus seal workers store all the data in the `~/lotusworker` folder (or `$LOTUS_WORKER_PATH` if defined).
+
+If you wish to change any of the storage locations of the Lotus miner, follow these steps:
+
+1. [Stop the miner](#safely-restarting-the-miner-daemon)
+2. Copy or move the data. You should move the data folder from the original location to the destination without changing any of the content.
+3. Edit `storage.json` with the new location.
+4. Start the miner.
+
+You can always [add additional storage locations to a Lotus miner](custom-storage-layout) with `lotus storage attach`.
+
+If you wish to change the storage location for any of the lotus workers:
+
+1. Stop the Lotus Worker.
+2. Move the data to the new location.
+3. Set `$LOTUS_WORKER_PATH` accordingly.
+4. Start the worker again.
+
+Any operations that the worker was performing before stopping will be restarted from the last checkpoint.
+
+:::warning
+Moving data between different workers is not currently supported. Moving the worker storage folder to a different worker machine will not work as the miner expects the ongoing sealing operations to be completed by the worker they were assigned to in the first place.
+:::
+
+## Using a different Lotus Node
+
+If you are planning to run maintenance on the Lotus node used by the miner, or if you need to failover to a different Lotus node because the current one does not work, follow these steps:
+
+1. [Stop the miner](#safely-restarting-the-miner-daemon)
+2. Set the `FULLNODE_API_INFO` environment variable to the location of the new node:
+
+```sh
+export FULLNODE_API_INFO=<api_token>:/ip4/<lotus_daemon_ip>/tcp/<lotus_daemon_port>/http
+```
+
+Follow these steps to learn [how to obtain a token](../../build/lotus/api-token-generation.md).
+
+3. If you have not exported your wallets yet, export them now from the old node and re-import them to the new Lotus node.
```

Comment:
I don't think lotus currently supports offline importing of keys, but I could be mistaken.

Reply:
This assumes both old and new node are running. Export/import instructions also mention just copying the `keystore` folder though, I'll make it clearer.